src/runtime/data/kata-collect-data.sh.in

```diff
@@ -310,16 +310,6 @@ show_runtime_log_details()
 	end_section
 }
 
-show_throttler_log_details()
```

Comment:
The changes on this commit look good. Apart from that, the commit ebbfa321ae132a02577240ab903d948c4646a267 introduced some handling for throttler that we should remove as well. Could you please try to `git-revert` that?

Reply:
I'm confused why the revert needs to occur. Does adding back that parameter do anything? The calls to find_system_journal_problems with the throttler are already removed.

Reply:
you are right @bookinabox , ignore my comment here.